councilmatic_core/models.py

```diff
@@ -531,7 +531,7 @@ def organization(self):
 
     @property
     def current_member(self):
-        if self.memberships:
```

Comment:
My impression from my earlier attempt to make councilmatic work without data was that this might have been a common oversight